### ISSUE TITLE
Azure OpenAI

### ISSUE THREAD

**Issue by stevenbaert** [assafelovic/gpt-researcher#168]
Can you make it compatible with Azure Open AI?
This should be helpfull: https://medium.com/@datatec.studio/guide-invoke-openai-api-with-python-azure-functions-1c128974c49

**Comment by pickettd** [comment]
Hi @stevenbaert - I just got this project working this morning with Azure OpenAI. I used the instructions from https://pypi.org/project/openai/ for Microsoft Azure endpoints along with the docs about the Langchain adapter here https://python.langchain.com/docs/guides/adapters/openai to get it to work.

When I was testing today, I had to provide the ChatCompletion.create function in llm_utils.py with a deployment_id parameter and open_api_key parameter and I configured Azure specific variables in config.py to get the adapter code in this repo to work with the Azure resources.
